.gitignore

```diff
@@ -15,6 +15,7 @@ Cargo.lock
 .DS_Store
 .idea
 /assets/*.tiled-session
+macros/target
```

Comment:
I think you can just add `target/` and that will ignore target directories anywhere in the hierarchy